docs/reference/buildx_create_for_multi_arch_build_in_k8s.md

```diff
@@ -0,0 +1,74 @@
+# Buildx create for multi-arch builds in k8s
+
+## With QEMU emulation
+
+```console
+$ KUBECONFIG=${KUBECONFIG} \
+    docker buildx create \
+        --name=builder \
+        --platform=linux/amd64,linux/arm64 \
+        --driver=kubernetes \
+        --driver-opt=namespace=buildkit,withqemu=true
+```
+
+### Known Issues
+
+QEUM only work well for `executing`, but for `compiling`, it will be very slow and may be panic.
+
+In the mode, if still want to compile binaries in docker.
+please use `FROM --platform=${BUILDPLATFORM}` to disable QEUM for compiling stage.
```

Comment:
I don't understand how `--platform=${BUILDPLATFORM}` relates to docker here

Reply:
Just like example below for golang.

When `--platform=${BUILDPLATFORM}` here, QEMU will not be used in the `builder` stage